examples/kubernetes-mixin-example/README.md

```diff
@@ -0,0 +1,46 @@
+# My First Bundle
```

Comment:
This file should be removed (since it's talking about my first bundle, and an empty bundle) or edited to explain the example